### PR TITLE
Update unified api imports

### PR DESCRIPTION
**Pull request by euaaaio**

### CODE REVIEW

**Comment by ai** [comment]
I already fixed Redux files :(

File: guide/starting/new-redux-client.md [hunk 1]
```diff
@@ -43,24 +43,24 @@ npm i react-redux redux
 Open `src/index.js`:
 
 ```diff
-  import * as serviceWorker from './serviceWorker';
-+ import { Provider } from 'react-redux';
-+ import reducer from './reducers';
-+ import { createStore } from 'redux';
+  import * as serviceWorker from './serviceWorker'
```
Comment (ai) [inline guide/starting/new-redux-client.md:46]: Create React App uses `;` by default

Comment (ai) [reply guide/starting/new-redux-client.md:46]: Maybe we should move to a better boilerplate? But lets for now keep the origin CRA code style

Comment (euaaaio) [reply guide/starting/new-redux-client.md:46]: 🤔 ok

Comment (euaaaio) [reply guide/starting/new-redux-client.md:46]: I removed it because you didn't use it in `replace-redux.md`

Comment (ai) [reply guide/starting/new-redux-client.md:46]: OK, let’s have `;`-less code

**Pull request merged**